src/common/options/rgw.yaml.in

```diff
@@ -3295,8 +3295,13 @@ options:
   type: uint
   level: advanced
   desc: Session token max duration
-  long_desc: Max duration in seconds for which the session token is valid.
+  long_desc: Max duration in seconds for which the session token is valid. This
```

Comment:
Max duration in seconds for which the temporary credentials returned by GetSessionToken is valid.

Reply:
Can you please correct the description for `rgw_sts_min_session_duration` also - This option can be used to configure the lower limit of durationSeconds of temporary credentials returned by  'AssumeRole*' calls.

Reply:
Done.